client/src/containers/TodoCardList/Item/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import TodoCard from 'src/components/TodoCardList/Item'
+import api from './requests'
+
+class TodoCardContainer extends Component {
+  state = {
+    completedAt: this.props.completedAt,
+  }
+
+  render = () => (
+    <TodoCard
+      description={this.props.description}
+      completedAt={this.state.completedAt}
+      onChange={this.updateCompletedAt}
+    />
+  );
+
+  updateCompletedAt = () => {
```

Comment:
@datarockets/cuties I think this container should update `Card` not `CompletedAt` resource

Reply:
I'd call it `toggleCompletion`

and implement like that:

```js
toggleCompletion = () =>
  this.isCompleted() ? this.uncomplete() : this.complete()
```

`updateCompletedAt` can be used like:
```js
updateCompletedAt = (date) =>
  this.setState({ completedAt: date })
```